karelics_vesc_can_driver/vesc_messages.py

```diff
@@ -476,7 +476,7 @@ def __init__(self, dutycycle=0):
 
     def get_encoded_msg(self):
         self.start_msg()
-        self.encode_int32(int(self.duty_cycle * 100000))
+        self.encode_int32(int(self.duty_cycle * 1e5))
```

Comment:
This is just the same, why do you want to change that?

Reply:
I just think it looks a bit cleaner. If not necessary I can just put values in the normal format and not the "e" format.